maintainers/maintainer-list.nix

```diff
@@ -1967,6 +1967,10 @@
     github = "leenaars";
     name = "Michiel Leenaars";
   };
+  leo60228 = {
+    github = "leo60228";
+    name = "leo60228";
+  };
```

Comment:
According to the head of this file an email address is mandatory.

Reply:
I misread it.